src/main/resources/fileTemplates/config.jelly.ft

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright DATE, Issuer.....
+
+Licensed under ...
+
+@Author
+-->
+
```

Comment:
```suggestion

```
I don't see a copyright in any of the Java templates. I believe this should be left out.
https://www.jetbrains.com/help/idea/copyright.html < this should handle insertion of copyright as far as I understand.